pywink/__init__.py

```diff
@@ -718,6 +720,140 @@ def _recent_state_set(self):
         return time.time() - self._last_call[0] < 15
 
 
+class WinkGarageDoor(WinkDevice):
+    """ represents a wink.py garage door
+    json_obj holds the json stat at init (and if there is a refresh it's updated
+    it's the native format for this objects methods
+    and looks like so:
+
+{
+  "data": {
+    "desired_state": {
+      "position": 0
+    },
+    "last_reading": {
+      "position_opened": "N\/A",
+      "position_opened_updated_at": 1450357467.371,
+      "tamper_detected_true": null,
+      "tamper_detected_true_updated_at": null,
+      "connection": true,
+      "connection_updated_at": 1450357538.2715,
+      "position": 0,
+      "position_updated_at": 1450357537.836,
+      "battery": null,
+      "battery_updated_at": null,
+      "fault": false,
+      "fault_updated_at": 1447976866.0784,
+      "disabled": null,
+      "disabled_updated_at": null,
+      "control_enabled": true,
+      "control_enabled_updated_at": 1447976866.0784,
+      "desired_position_updated_at": 1447976846.8869,
+      "connection_changed_at": 1444775470.5484,
+      "position_changed_at": 1450357537.836,
+      "control_enabled_changed_at": 1444775472.2474,
+      "fault_changed_at": 1444775472.2474,
+      "position_opened_changed_at": 1450357467.371,
+      "desired_position_changed_at": 1447976846.8869
+    },
+    "garage_door_id": "30528",
+    "name": "Garage Door",
+    "locale": "en_us",
+    "units": {
+
+    },
+    "created_at": 1444775470,
+    "hidden_at": null,
+    "capabilities": {
+      "home_security_device": true
+    },
+    "triggers": [
+
+    ],
+    "manufacturer_device_model": "chamberlain_garage_door_opener",
+    "manufacturer_device_id": "1133930",
+    "device_manufacturer": "chamberlain",
+    "model_name": "MyQ Garage Door Controller",
+    "upc_id": "26",
+    "upc_code": "012381109302",
+    "hub_id": null,
+    "local_id": null,
+    "radio_type": null,
+    "linked_service_id": "206203",
+    "lat_lng": [
+      0,
+      0
+    ],
+    "location": "",
+    "order": null
+  },
+  "errors": [
+
+  ],
+  "pagination": {
+
+  }
+}
+"""
+
+    def __init__(self, device_state_as_json, objectprefix="garage_doors"):
+        super(WinkGarageDoor, self).__init__(device_state_as_json,
+                                       objectprefix=objectprefix)
+        # Tuple (desired state, time)
+        self._last_call = (0, None)
+
+    def __repr__(self):
+        return "<Wink garage door %s %s %s>" % (self.name(), self.device_id(), self.state())
+
+    def state(self):
```

Comment:
``` state() ``` doesn't make sense here if ``` wait_till_desired_reached() ``` gets removed.

I'm just working under the assumption that you only included these methods because you thought they were supposed to be here :]